test/player_integration.js

```diff
@@ -592,6 +592,17 @@ describe('Player', () => {
       await waiter.waitUntilPlayheadReachesOrFailOnTimeout(video, 1, 10);
     });
 
+    it('with containerless formats', async () => {
+      player = new compiledShaka.Player(video);
+      // eslint-disable-next-line max-len
+      const url = 'https://storage.googleapis.com/shaka-demo-assets/raw-hls-audio-only/manifest.m3u8';
```

Comment:
Could we make this work with StreamGenerator?  If not, please use externalIt().  All tests that use external content are gated in that way.